wgpu/src/settings.rs

```diff
@@ -78,6 +78,7 @@ fn backend_from_env() -> Option<wgpu::BackendBit> {
             "dx11" => wgpu::BackendBit::DX11,
             "gl" => wgpu::BackendBit::GL,
             "webgpu" => wgpu::BackendBit::BROWSER_WEBGPU,
+            "primary" => wgpu::BackendBit::PRIMARY,
```

Comment:
Seems reasonable. Could you update the documentation comment too?

Reply:
I think I’ve updated the document. In addition to this, is it possible to implement another requirement, which is to get the AdapterInfo?Although it's not really a necessary feature, I'll see if there's a way to expose this method, and if not, forget it.

Reply:
>  In addition to this, is it possible to implement another requirement, which is to get the AdapterInfo?

Since `iced` is backend agnostic, we will need to figure out a way to do it without exposing the backend internals.